src/solver/yaml_helper.jl

```diff
@@ -1,24 +1,149 @@
 import YAML
+import ClimaAtmos as CA
 
 """
-    commands_from_yaml(yaml_file; filter_name=nothing)
-
-A vector of strings containing the `command`s in a (buildkite)
-yaml file, `yaml_file`. `filter_name` can be used to filter out
-required substrings from each elements of the vector.
-"""
-function commands_from_yaml(yaml_file; filter_name = nothing)
-    data = YAML.load_file(yaml_file)
-    steps = filter(x -> x isa Dict && haskey(x, "steps"), data["steps"])
-    cmds = map(x -> x["steps"], steps)
-    cmds = vcat(cmds...)
-    cmds = map(x -> x["command"], cmds)
-    cmds = vcat(cmds...)
-    cmds = map(x -> split(x, "\n"), cmds) # handle commands that run multiple scripts
-    cmds = vcat(cmds...)
-    cmds = map(x -> x isa AbstractString ? x : join(x), cmds)
-    if filter_name ≠ nothing
-        filter!(x -> occursin(filter_name, x), cmds)
+    default_config_dict()
+
+Loads the default configuration from files into a Dict for use in AtmosConfig().
+"""
+function default_config_dict()
+    default_config_file =
+        joinpath(pkgdir(CA), ".buildkite", "default_config.yml")
+    config = YAML.load_file(default_config_file)
+    edmf_config_file =
+        joinpath(pkgdir(CA), ".buildkite", "default_edmf_config.yml")
+    edmf_config = YAML.load_file(edmf_config_file)
+    # Combine base config with EDMF config - Don't allow duplicate entries.
+    isempty(intersect(keys(config), keys(edmf_config))) ||
+        error("Duplicate keys in default config and EDMF config.")
+    merge!(config, edmf_config)
+    # Strip out help messages
+    for (k, v) in config
+        config[k] = v["value"]
+    end
+    return config
+end
+
+"""
+    config_from_target_job(target_job)
+
+Given a job id string, returns the configuration for that job.
+Does not include the default configuration dictionary.
+"""
+function config_from_target_job(target_job)
+    for entry in configs_per_job_id(joinpath(pkgdir(CA), ".buildkite"))
```

Comment:
Use of `CA` (see other comments about `import ClimaAtmos as CA` from within `src/`).